test/cookbooks/selinux_policy_test/recipes/fcontext.rb

```diff
@@ -40,7 +52,6 @@
   notifies :run, 'ruby_block[fail-mismatch]', :immediate
 end
 
-# Should not run again
```

Comment:
It actually should, that's why I deleted that comment. I guess it's a remnant from a copy/paste you made before.
And will do.